autora/nina-solon/css/style.css

```diff
@@ -0,0 +1,345 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 15px;
+    color: #63458A;
+}
+
+a {
+    text-decoration: none;
+}
+
+h2 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 1.3rem;
+    text-align: center;
+    margin-bottom: 5vh;
+}
+
+p {
+    line-height: 150%;
+    margin-bottom: 3vh;
+}
+
+.navegacao {
+    width: 100%;
+    height: 12vh;
+    background-color: #674582;
+    position: fixed;
+    top: 0;
+    display: flex;
+    justify-content: center;
+    z-index: 100;
+}
+
+@media (max-widht: 768px) {
+    .navegacao {
+    height: 10vh;
+    }
+}
+
+.navegacao__conteudo {
+    width: 100%;
+    max-width: 960px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.logo {
+    width: 200px;
+}
+
+.navegacao__menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 0.7rem;
+    text-transform: uppercase;
+}
+
+.menu-reprograma {
+    margin-right: 3vw;
```

Comment:
Parabéns pelas unidades de medida usadas, ajudam quando for fazer o responsivo, vou adotar :D